static/css/restyle/restyle.less

```diff
@@ -1415,6 +1419,10 @@ h3.author .transfer-ownership {
   padding: 30px 0;
   width: 100vw;
 
+  .html-rtl& {
```

Comment:
This one has no space but the other one does; why is that? (forgive my lack of LESS-foo)